modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3259,6 +3259,9 @@ void GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode *p_call, bool p_is_a
 		GDScriptParser::IdentifierNode *base_id = nullptr;
 		if (subscript->base->type == GDScriptParser::Node::IDENTIFIER) {
 			base_id = static_cast<GDScriptParser::IdentifierNode *>(subscript->base);
+			if (base_id->name == "Globals") {
+				is_global = true;
+			}
```

Comment:
This is incorrect, see for example:

https://github.com/godotengine/godot/blob/fd33c7b32f54e9ac3d346be718618575338cd7ef/modules/gdscript/gdscript_analyzer.cpp#L3975-L3977

But I'm not sure that the linked issue is actually a bug.